Networking/Networking/Model/Stats/OrderStatsV4Totals.swift

```diff
@@ -0,0 +1,102 @@
+        self.grossRevenue = grossRevenue
+        self.couponDiscount = couponDiscount
+        self.coupons = coupons
+        self.refunds = refunds
+        self.taxes = taxes
+        self.shipping = shipping
+        self.netRevenue = netRevenue
+        self.products = products
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let orders = try container.decode(Int.self, forKey: .ordersCount)
+        let itemsSold = try container.decode(Int.self, forKey: .itemsSold)
+        let grossRevenue = try container.decode(Decimal.self, forKey: .grossRevenue)
+        let couponDiscount = try container.decode(Decimal.self, forKey: .couponDiscount)
+        let coupons = try container.decode(Int.self, forKey: .coupons)
+        let refunds = try container.decode(Decimal.self, forKey: .refunds)
+        let taxes = try container.decode(Decimal.self, forKey: .taxes)
+        let shipping = try container.decode(Decimal.self, forKey: .shipping)
+        let netRevenue = try container.decode(Decimal.self, forKey: .netRevenue)
+        let products = try container.decodeIfPresent(Int.self, forKey: .products)
+
+        self.init(orders: orders,
+                  itemsSold: itemsSold,
+                  grossRevenue: grossRevenue,
+                  couponDiscount: couponDiscount,
+                  coupons: coupons,
+                  refunds: refunds,
+                  taxes: taxes,
+                  shipping: shipping,
+                  netRevenue: netRevenue,
+                  products: products)
+    }
+}
+
+
+// MARK: - Conformance to Equatable
+//
+extension OrderStatsV4Totals: Equatable {
+    public static func == (lhs: OrderStatsV4Totals, rhs: OrderStatsV4Totals) -> Bool {
+        return lhs.orders == rhs.orders &&
+            lhs.itemsSold == rhs.itemsSold &&
+            lhs.grossRevenue == rhs.grossRevenue &&
+            lhs.couponDiscount == rhs.couponDiscount &&
+            lhs.coupons == rhs.coupons &&
+            lhs.refunds == rhs.refunds &&
+            lhs.taxes == rhs.taxes &&
+            lhs.shipping == rhs.shipping &&
+            lhs.netRevenue == rhs.netRevenue &&
+            lhs.products == rhs.products
+    }
+
+    public static func < (lhs: OrderStatsV4Totals, rhs: OrderStatsV4Totals) -> Bool {
+        return lhs.grossRevenue < rhs.grossRevenue ||
+            (lhs.grossRevenue == rhs.grossRevenue && lhs.orders < rhs.orders)
+    }
+}
+
+
+// MARK: - Constants!
+//
+private extension OrderStatsV4Totals {
+    enum CodingKeys: String, CodingKey {
+        case ordersCount = "orders_count"
+        case itemsSold = "num_items_sold"
+        case grossRevenue = "gross_revenue"
+        case couponDiscount = "coupons"
+        case coupons = "coupons_count"
+        case refunds = "refunds"
+        case taxes = "taxes"
+        case shipping = "shipping"
+        case netRevenue = "net_revenue"
+        case products = "products"
```

Comment:
if I remember correctly, if the value is the same as the case name, then it doesn't have to be specified (like the example in the [Apple doc](https://developer.apple.com/documentation/foundation/archives_and_serialization/encoding_and_decoding_custom_types)) - do we usually still spell out the value for consistency/accuracy (like the chance of typo twice < once)?

Reply:
True. I don't think we spell those out anywhere else, so I'll remove them

Reply:
I think it was done to visually balance out the code. It's harder to skim when it's a mix.